scripts/cluster/yarn_cluster_setup/README.md

```diff
@@ -0,0 +1,62 @@
+
+Setup an experiment on Cloudlab using the SparkFHE-Base-Ubuntu18.04 image. Please make note of the master node login.
+
+Also, note that the scripts are designed to run on Master Node.
+
+## Prerequisites
+* On all the nodes follow [instructions](https://github.com/SpiRITlab/SparkFHE-Examples/wiki) to setup SparkFHE distribution. Make sure you have gone through all the steps and the mySparkSubmitLocal.bash script runs correctly.
+
+# Move files to Master Node(TEMPORARY STEP FOR THIS BRANCH)
+
+The script move_files_to_master.sh moves scripts from local machine to Master Node. The name of the master node needs to be passed as an argumnent to the script.
+```
+bash move_files_to_master.sh username@id.region.cloudlab.us
+```
+
+# SSH into Master Node
+SSH into address for master node and navigate to the address /yarn_spark_cluster_setup
+```
+ssh -p 22 username@id.region.cloudlab.us
+cd /spark-3.0.0-SNAPSHOT-bin-SparkFHE/SparkFHE-Addon/scripts/cluster/yarn_cluster_setup
+```
+
+# Install Hadoop and Spark on all nodes through Master Node
+Specify the names of nodes as arguments.
+```
+sudo bash install.sh master worker1 worker2 ...
+```
+This script moves files from master to other nodes and installs pre-requisites. After this script is integrated with master branch, the code for moving files to other nodes can be commented out. When TestDrive.bash is run on all nodes, the scripts to install and setup Yarn cluster should be available on all nodes.
+
+# Start Yarn Spark Cluster on Master
+```
+cd Test_Pi
+sudo bash start_yarn_spark_cluster.sh
+sudo bash run_spark_job.sh
+```
+Use the link generated after successful completion of cluster building to view the web interface for Yarn. Other links can be generated by changing the port number.
+
+YARN Interface:
+
+http://master.experiment-name.iotx-pg0.utah.cloudlab.us:8088/cluster
```

Comment:
utah.cloudlab.us is because your cluster is in utah. It will be different in another cloudlab network.

Reply:
I have replaced utah with regionname in the latest commit